CONFIGDOC.md

```diff
@@ -69,9 +69,9 @@ You can specify build options for your MATLAB build by first selecting **Build o
 
 MATLAB exits with exit code 0 if the specified tasks run without error. Otherwise, MATLAB terminates with a nonzero exit code, which causes the step to fail.
 
-Starting in R2024a, you can access the result of running a MATLAB build interactively in your Jenkins interface. After your build runs, the Jenkins build summary page displays the number of tasks that ran, failed, and were skipped. You can click the **MATLAB Build Results** link on the page to access the table of task results. The table provides information about each task that was part of the MATLAB build. Clicking a task name in the table displays relevant build log information on the **Console Output** page.
+Starting in R2024a, you can access the result of running a MATLAB build interactively in your Jenkins interface. After your build runs, the Jenkins build summary page displays the number of tasks that ran, failed, and were skipped. You can click the **MATLAB Build Results** link on the page to access the table of task results. The table provides information about each task that was part of the MATLAB build. For example, if a task was skipped, the table displays the reason that the task was skipped. Clicking a task name in the table displays relevant build log information on the **Console Output** page.

```

Comment:
"Clicking a task name in the table displays relevant build log information on the **Console Output** page." -> "Clicking a task name in the table navigates to the relevant section of the build log on the **Console Output** page" 

to make it clear that we're actually just changing pages rather than pulling information from that location onto the current page?

Reply:
The current text doesn't sound confusing. I will run this comment by the editor and let her decide.

Reply:
Ah gotcha. I found it a bit confusing but not a big deal